pages/TheExplorationEditor.html

```diff
@@ -86,7 +86,7 @@ <h4>
     <a href="https://github.com/oppia/oppia">open source codebase</a>.
   </p>
 
-  <img src="static/images/interactionSelector.png" width="100%">
+  <img src="static/images/interactionSelector.png" width="100%" alt="Highlighted in red are areas of the interaction page, showing the interaction categories and the interaction types.">
```

Comment:
A screenshot of XXX. The interaction categories and the interaction types are highlighted in red.